paasta_tools/check_services_replication_tools.py

```diff
@@ -54,6 +74,22 @@ def parse_args() -> argparse.Namespace:
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+    parser.add_argument(
+        "--warn",
+        dest="under_replicated_warn_pct",
+        type=float,
+        default=5,
+        help="The percentage of under replicated service instances past which "
+        "the script will return a warning status",
+    )
+    parser.add_argument(
+        "--crit",
+        dest="under_replicated_crit_pct",
+        type=float,
+        default=10,
+        help="The percentage of under replicated service instances past which "
+        "the script will return a critical status",
+    )
```

Comment:
I would be curious to hear opinions on sane defaults for these.

Reply:
idk, 30% / 50%? Probably depends on the service. Would it be annoying for it to be configurable?

Reply:
to clarify, this check is for _all_ instances on a cluster.  So it's a question of what percentage of service instances are failing their replication checks.